### ISSUE TITLE
Cannot find module '.\src\lzma_worker.js'

### ISSUE THREAD

**Issue by gauravnayak210** [nmrugg/LZMA-JS#20]
I am using this module to decompress the already LZMA compressed data, I have installed it as you guys mentioned in doc.

But, When I do, `node app`, I am getting uncaught exception, like,
`Uncaught Exception...
Error: Cannot find module '.\src\lzma_worker.js'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at load_lzma (***\foo\node_modules\lzma\index.js:10:12)
    at Object.<anonymous> (***\foo\node_modules\lzma\index.js:14:8)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)`

so, I checked `index.js` file in LZMA module and I found that,
This line,

`return require("." + p.sep + "src" + p.sep + "lzma_worker.js").LZMA_WORKER;`

cause an error in `load_lzma()` function (It may because value of `p.sep` is `\`, but its not working in my case)

And when I changed it as

`return require("./src/lzma_worker.js").LZMA_WORKER;`

then, It works perfectly.

Basically, I do not want to change `index.js` of LZMA module file, as it might be updated later by you guys.

kindly reply.

**Comment by nmrugg** [comment]
Thanks for the report! Should be fixed now.
